todo_app.js

```diff
@@ -63,15 +75,28 @@
          }*/
     }
 
-    function renderTodo(todoObject) {
+    function renderTodos(list) {
         // return string: <li><input type="checkbox" checked or
         // unckecked if todoObject.checked is true> todo text</li>
+        var lsTodos = [];
```

Comment:
You don't need to assign empty array because lsTodos will be redefined on next line.

`var todosList = TodoManager.loadTodos();`